src/askui_runner/project_template/helper/jest.setup.ts.jinja

```diff
@@ -27,7 +33,10 @@ beforeEach(async () => {
 afterEach(async () => {
   await aui.stopVideoRecording();
   const video = await aui.readVideoRecording();
-  AskUIAllureStepReporter.attachVideo(video);
+  const size = getByteSizeInMB(video);
+  if (size < 9.9) {
```

Comment:
I would move this into its own function and explain that this is only a workaround for the aws api gateway limit with function name and comment(s), e.g., something like

```typescript
const AWS_API_GATEWAY_MAX_PAYLOAD_SIZE_IN_MB = 10

function isVideoTooLargeToBeUploadedThroughAwsApiGateway(video: string): boolean {
   ...
}
```